sw/device/lib/dif/dif_rv_core_ibex.c

```diff
@@ -361,3 +361,31 @@ dif_result_t dif_rv_core_ibex_trigger_sw_fatal_err_alert(
                       RV_CORE_IBEX_SW_FATAL_ERR_REG_OFFSET, reg);
   return kDifOk;
 }
+
+enum {
+  kIbexCrashDumpBytesCount = sizeof(dif_rv_core_ibex_crash_dump_info_t),
+  kIbexCrashDumpWordsCount = kIbexCrashDumpBytesCount / sizeof(uint32_t),
```

Comment:
Probably need a static assert to ensure that `kIbexCrashDumpBytesCount` is a multiple of `sizeof(uint32_t)`. Same for other constants that embed such assumption.

Reply:
done